tensorflow/python/summary/summary_test.py

```diff
@@ -100,7 +100,8 @@ def testImageSummaryWithFamily(self):
     with self.cached_session() as s:
       i = array_ops.ones((5, 2, 3, 1))
       with ops.name_scope('outer'):
-        im = summary_lib.image('inner', i, max_outputs=3, family='family')
+        im = summary_lib.image('inner', i, max_outputs=3, vmin=1.0, vmax=1.0,
```

Comment:
You should also test the`vmin < vmax`, `vmin` only, and `vmax` only cases.

Reply:
Have added more test cases.